### PR TITLE
introduces support for shortDescription

### PR DESCRIPTION
**Pull request by jonathansampson**
addresses https://github.com/facundoolano/google-play-scraper/issues/687

### CODE REVIEW

**Comment by PierfrancescoSoffritti** [comment]
Thanks for the prompt PR!

nit: you might wanna add a mention of the short description to [the example in the README](https://github.com/facundoolano/google-play-scraper?tab=readme-ov-file#app) as well :)

**Comment by jonathansampson** [comment]
Good call, @PierfrancescoSoffritti.

File: lib/app.js [hunk 1]
```diff
@@ -49,6 +49,7 @@ const MAPPINGS = {
     path: ['ds:5', 1, 2],
     fun: helper.descriptionHtmlLocalized
   },
+  shortDescription: ['ds:5', 1, 2, 73, 0, 1],
```
Comment (facundoolano) [inline lib/app.js:52]: Isn't this just the same as the summary below?

Comment (jonathansampson) [reply lib/app.js:52]: Whoa. It is. Sorry, @PierfrancescoSoffritti, it looks like `summary` has what you're seeking :)

**Pull request closed without merging**